conditional/blueprints/co_op.py

```diff
@@ -42,6 +42,8 @@ def submit_co_op_form(user_dict=None):
     semester = post_data['semester']
     if post_data['semester'] not in valid_semesters:
         return "Invalid semester submitted", 400
+    if not ldap_is_current_student(user_dict['account']):
+        return "must be current student", 403
```

Comment:
```suggestion
        return "Must be current student", 403
```
You capitalize `Invalid` in the error above, so I would capitalize Must here